fabric-api-dev/src/main/java/net/fabricmc/fabric/impl/FabricDevProperties.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl;
+
+import java.util.function.Supplier;
+
+import com.mojang.brigadier.ParseResults;
+
+import net.minecraft.Bootstrap;
+import net.minecraft.server.command.CommandManager;
+import net.minecraft.server.dedicated.EulaReader;
+import net.minecraft.util.collection.Weight;
+import net.minecraft.world.Heightmap;
+
+import net.fabricmc.loader.api.FabricLoader;
+
+@SuppressWarnings("JavadocReference")
+public class FabricDevProperties {
+	/**
+	 * Logs an error when a weight is set to zero</br>
```

Comment:
FYI: These java docs are all in impl classes, so wont end up in the published javadocs. They should be documented elsewhere.